### ISSUE TITLE
After upgrading to Ubuntu 19.04: Error loading extension

### ISSUE THREAD

**Issue by kleinfreund** [maoschanz/emoji-selector-for-gnome#81]
After upgrading to Ubuntu 19.04, I get a small warning symbol in the Tweaks tool’s extension page. Upon hover, it reads "Error loading extension".

Pressing <kbd>Alt</kbd>+<kbd>F2</kbd>, entering `lg`, pressing <kbd>Enter</kbd>, clicking “Extensions”, clicking “Show errors” reveals the following error:

```
TypeError: this._construct is not a function
```

Running the following claims that emoji-selector is already enabled, but it isn’t.

```sh
gnome-shell-extension-tool -e emoji-selector@maestroschan.fr
```

**Comment by maoschanz** [comment]
>but it isn’t.

It is: an extension enabled with errors is still an enabled extension.

Could you provide the version of the extension ? Written in the "about" page of its preferences

**Comment by kleinfreund** [comment]
> Could you provide the version of the extension ?

v11

I made sure and re-downloaded the extension from the latest GitHub release.

**Comment by kleinfreund** [comment]
I found this page: https://www.khattam.info/solved-gnome-shell-extensions-not-loading-after-update-2011-05-29.html

Basically, it recommends adding the Gnome Shell version to the extension’s metadata.json file:

```json
{
  "description": "This extension provides a parametrable popup menu with some emojis, clicking on an emoji copies it to the clipboard. An appropriate font like 'Twitter Color Emoji' or 'EmojiOne Color' should be installed on your system for a better visual result. Color emojis are available only for GNOME Shell 3.26 and more.",
  "name": "Emoji Selector",
  "shell-version": [
    "3.14",
    "3.16",
    "3.18",
    "3.20",
    "3.22",
    "3.24",
    "3.26",
    "3.28",
    "3.32.1"
  ],
  "url": "https://github.com/maoschanz/emoji-selector-for-gnome",
  "uuid": "emoji-selector@maestroschan.fr",
  "gettext-domain": "emoji-selector",
  "settings-schema": "org.gnome.shell.extensions.emoji-selector",
  "version": 11
}
```

After logging out and back in, the extension loads fine and works. I’m not sure if that is actually what fixed this. :/

**Comment by maoschanz** [comment]
So, quick explanation:

- **I** write the metadata.json file with compatible versions. If a version is not listed in this file, it means it's not compatible, and **it's not intended to be compatible** at all. 
- Manually adding versions will not change anything, whatever a 8 years old blog post tries to invent. This file is declarative, and allows extensions.gnome.org to give you only extensions compatible with your system, in order to avoid errors. Lying in this file will never debug errors.
- https://extensions.gnome.org/extension/1162/emoji-selector/ will give you the version 15, which is intended to be compatible with GNOME Shell 3.32
